system-tests/_support/index.js

```diff
@@ -219,23 +219,27 @@ Cypress.addChildCommand("triggerHover", function(elements) {
 });
 
 /**
- * Launches a new service using the dcos CLI
+ * Launches a new service in group TEST_UUID using the dcos CLI
  *
- * @param {String} serviceName - The service name which will launch
- * @param {String} serviceJsonPath - The path to JSON file for service definition
+ * @param {Object} serviceDefinition - The service JSON definition file
  *
  */
-export function createService(serviceName, serviceJsonPath) {
+export function createService(serviceDefinition) {
   cy.exec(
-    `sed 's/${serviceName}/${Cypress.env("TEST_UUID")}\\/${serviceName}/g' ${serviceJsonPath} | dcos marathon app add`
+    `echo '${JSON.stringify(serviceDefinition)}' | dcos marathon app add`
```

Comment:
Wouldn't it only create the service in this group if the `id` inside `serviceDefinition` started with `TEST_UUID`? Feel like the description is misleading because it depends on the input. Is there a better way to do this? L231 is also relying on TEST_UUID being in the `id` field for it to work. Let's rethink this function.